one_fm/one_fm/doctype/hr_and_payroll_additional_settings/hr_and_payroll_additional_settings.json

```diff
@@ -279,26 +277,12 @@
    "fieldname": "section_break_nkbfu",
    "fieldtype": "Section Break",
    "hide_border": 1
-  },
```

Comment:
@AbrahamAdekunle why you removed this fields

Reply:
@pjamsheer , resolved